File: HouseM8-Flutter/housem8_flutter/lib/models/message.dart
class Message {
  final int id;
  final int chatId;
  final String messageSend;
  final int senderId;
  final DateTime time;

  Message({this.id, this.chatId, this.messageSend, this.senderId, this.time});

  factory Message.fromJson(Map<String, dynamic> json) {
    return Message(
      id: int.parse(json["id"].toString()),
      chatId: int.parse(json["chatId"].toString()),
      messageSend: json["messageSend"],
      senderId: int.parse(json["senderId"].toString()),
      time: DateTime.parse(json["time"].toString()),
    );
  }
}

File: HouseM8API/HouseM8API/Models/Chat/Message.cs
using System;

namespace HouseM8API.Models
{
    public class Message
    {
        public int Id { get; set; }
        public int ChatId { get; set; }
        public string MessageSend { get; set; }
        public int SenderId { get; set; }
        public DateTime Time { get; set; }
    }
}
